package.json

```diff
@@ -249,6 +249,7 @@
     "re2": "1.19.1"
   },
   "devDependencies": {
+    "@hyrious/marshal": "0.2.5",
```

Comment:
Later we maybe can use this library to both serialization (tests) and deserialization (prod)